packages/biniou/biniou.1.0.5/url

```diff
@@ -1,2 +1,2 @@
-archive: "https://mjambon.github.io/mjambon2016/releases/biniou/biniou-1.0.5.tar.gz"
+archive: "https://github.com/mjambon/biniou/archive/v1.0.5.tar.gz"
```

Comment:
because they were broken :-) They're being rearranged as Martin is retiring his old site.